### DISCUSSION TITLE
Are there any workarounds for auto-approving automerge PRs on GitHub Enterprise?

### DISCUSSION THREAD

**Discussion by bavedarnow** [renovatebot/renovate#8264] [Request Help]
Since we require a certain amount of approvals on PRs, the automerge feature wont work on GHE. [This link](https://docs.renovatebot.com/automerge-configuration/#required-pull-request-reviews) discusses using the renovate-approve apps. Does anyone know of a solution for GitHub Enterprise? I wasn't able to find a bot that we could run ourselves after the renovate job.

**Comment by rarkins** [comment]
You need somewhere to host it which can receive webhooks from your GHE. Do you run renovate in Docker?

**Reply by bavedarnow** [reply]
Yeah, we run it in a docker container in Jenkins

**Reply by rarkins** [reply]
You would need a different approach to run the renovate approve bot, because it needs to listen to webhooks and react - therefore needs to be stateful/long-running and not start-run-stop like it sounds like you have.
